discovery-provider/Caddyfile

```diff
@@ -15,4 +15,5 @@ route /dashboard* {
     file_server
 }
 
-reverse_proxy backend:5000
+# defaults to backend:5000
+reverse_proxy {$ROOT_HOST}
```

Comment:
ROOT_HOST gets read in through `override.env` as opposed to the original .env implementation, see PR description for how to add that